src/Tabs/Tabs.js

```diff
@@ -233,11 +239,13 @@ class Tabs extends Component {
 
     return (
       <div style={prepareStyles(Object.assign({}, style))} {...other}>
-        <div style={prepareStyles(Object.assign(styles.tabItemContainer, tabItemContainerStyle))}>
-          {tabs}
-        </div>
-        <div style={{width: inkBarContainerWidth}}>
-          {inkBar}
+        <div className={labelsContainerClassName}>
```

Comment:
Changing the DOM structure, to be able to add Sticky feature or any other Dynamic UI functions to the tabs labels area.